.github/workflows/test.yml

```diff
@@ -162,13 +162,28 @@ jobs:
         with:
           python-version: 3.8
       - uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: maturin-test-docker-${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            maturin-test-docker-${{ runner.os }}-buildx-
       - name: Build
         uses: docker/build-push-action@v2
         with:
           push: false
           tags: maturin
           load: true
-          cache-from: type=registry,ref=konstin2/maturin:buildcache
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
```

Comment:
Does this still need `mode=max` to still work after code changes?

Reply:
I am not sure, though I've never encountered any issue in my past experience with this approach.

Reply:
I'll push a minor code change to test it.

Reply:
It now rebuilds all dependencies, with `mode=max` it should only rebuild maturin itself